analyses/infercnv-consensus-cell-type/scripts/build-normal-reference/build-reference-SCPCP000015.R

```diff
@@ -87,8 +92,27 @@ immune_subset_cells_df <- immune_cells_df |>
     consensus_annotation == "macrophage" |
       stringr::str_detect(consensus_annotation, "T cell")
   )
+
+# Subset to the endothelial cells only
+endo_cells_df <- consensus_df |>
+  dplyr::filter(
+    consensus_annotation == "endothelial cell",
```

Comment:
We probably want to include all types of endothelial cells. Looking at the possible consensus cell types, that should be `blood vessel endothelial cell` and `microvascular endothelial cell` along with `endothelial cell`. I used this table to get that information: https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/analyses/cell-type-consensus/references/consensus-validation-groups.tsv